### PR TITLE
#251 - started to implement administration panel in order to be able …

### PR DESCRIPTION
**Pull request by aviklai**
…to do send email to all subscribers and do various other admin functions.

### CODE REVIEW

File: templates/admin/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% extends 'admin/master.html' %}
+{% block body %}
+{{ super() }}
+<div class="row-fluid">
+
+    <div>
+        {% if current_user.is_authenticated() %}
+        <h1>Welcome to Anyway Administration Panel</h1>
```
Comment (danielhers) [inline templates/admin/index.html]: Gal prefers "ANYWAY" (all caps) :)

File: main.py [hunk 2]
```diff
@@ -189,6 +192,125 @@ def updatebyemail():
...
+        return redirect(url_for('.index'))
+
+class SendToSubscribersView(BaseView):
+    @login.login_required
+    @expose('/')
+    def index(self):
+        return self.render('sendemail.html')
+
+    def is_visible(self):
+        if login.current_user.is_authenticated():
```
Comment (danielhers) [inline main.py]: Just return the boolean itself instead of an `if`

File: main.py [hunk 3]
```diff
@@ -189,6 +192,125 @@ def updatebyemail():
...
+    login_manager = login.LoginManager()
+    login_manager.init_app(app)
+
+    # Create user loader function
+    @login_manager.user_loader
+    def load_user(user_id):
+        return db_session.query(User).get(user_id)
+
+
+class MyModelView(sqla.ModelView):
```
Comment (danielhers) [inline main.py]: Why "My" (MyModelView, MyAdminIndexView, my_master)? Is that the standard in Flask apps?

**Comment by danielhers** [comment]
Nice!
1. Where is the code that actually sends the email to the subscribers?
2. I see you commented out the registration code. So we'll have to register the admin user(s) through the DB?
3. I tried the code (on [Heroku](https://anyway-pr-253.herokuapp.com/)) and it got a server error because it couldn't find `wtforms`. Maybe you need to add it to `requirements.txt`?

**Comment by aviklai** [comment]
O.K, I did the changes.
About the three last questions:
1. still not ready :)  I just wanted to see that this part is ok and that it integrates well with the site.
The sending of the email is not supposed to be a problem. I plan to use some flask smtp library like:
https://pythonhosted.org/flask-mail/
or:
https://flask-email.readthedocs.org/en/latest/
I will just need to query the database, check who is subscribed and put all of their mail in the BCC of the smtp email. by the way - do we have an email that supports smtp? I can use dummy email for the tests though.

2. Yes, I planned to to that through the DB and the main reason I left this code is because I want the password to be stored as a hashed password so I need the this function to hash it:
generate_password_hash(form.password.data)

3. I added WTForms, Flask-Login and Flask-Admin to the requirements file.

**Comment by danielhers** [comment]
Great, to complete #251 we just need the things you listed here :)

**Pull request merged**